cli/test/test_common.py

```diff
@@ -0,0 +1,54 @@
+import os
```

Comment:
Instead of defining functions, we can have define an abstract `PopperTest` class that has these as members. All the tests that we write as part of this PR can derive from this class, so that common functionality gets consolidated

Reply:
Yes @ivotron,  This would be a much better approach to create this file.